archive/archive_ingest/src/test_archive_ingest.py

```diff
@@ -0,0 +1,48 @@
+# -*- encoding: utf-8 -*-
+
+import os
+import archive_ingest
+import pytest
+
+
+def test_sends_location_to_sns(sns_client, topic_arn):
+    os.environ['INGEST_TOPIC_ARN'] = topic_arn
```

Comment:
minor: prefer `os.environ.update({'INGEST_TOPIC_ARN': topic_arn})`, as that doesn't blat existing environment config.

Also, the `topic_arn` fixture auto-populates the `TOPIC_ARN` environment variable, if you wanted to be consistent with our other Lambdas.